specifications/api/swagger.yaml

```diff
@@ -1193,6 +1237,20 @@ components:
         pledge: *stakePoolPledge
         metadata: *stakePoolMetadata
         retirement: *stakePoolRetirement
+        delisted:
+          type: boolean
+
+    ApiListStakePools: &ApiListStakePools
+      type: object
+      required:
+        - pools
+        - gc_status
+      properties:
+        gc_status:
+          <<: *ApiGCStatus
+        pools:
```

Comment:
This is an API breaking change. Since the GC status is already available on its own on an endpoint, I believe this breaking change can be avoided.

Reply:
Yes it is and I believe this was discussed.